packages/react-magma-dom/src/components/Tabs/Tabs.stories.tsx

```diff
@@ -201,6 +217,107 @@ export const Scrolling = ScrollingTemplate.bind({});
+      dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+      proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+    </p>
+    <p>
+      Amet aliquam id diam maecenas ultricies mi. Venenatis tellus in metus
+      vulputate eu scelerisque felis imperdiet. Tristique sollicitudin nibh sit
+      amet commodo nulla facilisi nullam. Facilisis sed odio morbi quis commodo
+      odio aenean. Odio tempor orci dapibus ultrices in iaculis nunc sed augue.
+      In arcu cursus euismod quis viverra nibh cras. Tincidunt ornare massa eget
+      egestas purus viverra accumsan in nisl. Porta nibh venenatis cras sed
+      felis. Felis donec et odio pellentesque diam. Aliquam ut porttitor leo a
+      diam sollicitudin. Sed sed risus pretium quam vulputate dignissim
+      suspendisse in. Fringilla ut morbi tincidunt augue interdum. Vel elit
+      scelerisque mauris pellentesque pulvinar.
+    </p>
+    <p>
+      Duis aute irure dolor in reprehenderit in voluptate velit esse cillum
+      dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+      proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+    </p>
+    <p>
+      Amet aliquam id diam maecenas ultricies mi. Venenatis tellus in metus
+      vulputate eu scelerisque felis imperdiet. Tristique sollicitudin nibh sit
+      amet commodo nulla facilisi nullam. Facilisis sed odio morbi quis commodo
+      odio aenean. Odio tempor orci dapibus ultrices in iaculis nunc sed augue.
+      In arcu cursus euismod quis viverra nibh cras. Tincidunt ornare massa eget
+      egestas purus viverra accumsan in nisl. Porta nibh venenatis cras sed
+      felis. Felis donec et odio pellentesque diam. Aliquam ut porttitor leo a
+      diam sollicitudin. Sed sed risus pretium quam vulputate dignissim
+      suspendisse in. Fringilla ut morbi tincidunt augue interdum. Vel elit
+      scelerisque mauris pellentesque pulvinar.
+    </p>
+  </>
+);
+
+const ScrollSpyTemplate: Story<TabsProps> = args => (
+  <TabsScrollSpyContainer isInverse={args.isInverse}>
+    <TabScrollSpyPanel tabLabel="Card 1">
+      <Heading level={4}>Area 1</Heading>
+      {scrollContent}
+    </TabScrollSpyPanel>
+    <TabScrollSpyPanel tabLabel="Card 2">
+      <Heading level={4}>Area 2</Heading>
+      {scrollContent}
+    </TabScrollSpyPanel>
+    <TabScrollSpyPanel tabLabel="Card 3">
+      <Heading level={4}>Area 3</Heading>
+      {scrollContent}
+    </TabScrollSpyPanel>
+    <TabScrollSpyPanel tabLabel="Card 4">
+      <Heading level={4}>Area 4</Heading>
+      {scrollContent}
+    </TabScrollSpyPanel>
+    <TabScrollSpyPanel tabLabel="Card 5">
+      <Heading level={4}>Area 5</Heading>
+      {scrollContent}
+    </TabScrollSpyPanel>
+  </TabsScrollSpyContainer>
+);
+
+export const ScrollSpy = ScrollSpyTemplate.bind({});
+ScrollSpy.args = {
+  ...Default.args,
+};
+ScrollSpy.parameters = { ...Default.parameters };
```

Comment:
We should exclude the controls that don't affect scroll spy
```suggestion
ScrollSpy.parameters = { controls: { exclude: ['iconPosition', 'alignment', 'borderPosition', 'orientation'] } };
```